### DISCUSSION TITLE
3d classification with multi output head

### DISCUSSION THREAD

**Discussion by YingdiZhang** [Project-MONAI/MONAI#6876] [Q&A]
Thanks for the great job. I am using monai in a 3D classification task. I want to classify a CT scan in two ways, the first is binary classification of if the disease exist, the second is classify another disease into not exist, low level or high level.  I want to achieve this classification task via multi output head, how to achieve that?

**Comment by KumoLiu** [comment]
Hi @YingdiZhang,

You can create your own custom network that includes multiple heads.

Here is an example in MONAI which outputs three heads. You can refer to it.
https://github.com/Project-MONAI/MONAI/blob/2daabf917a1e6afbc738d22db602f4859672a90d/monai/networks/nets/hovernet.py#L407

Hope it helps, thanks!
